### DISCUSSION TITLE
update sinclair/typebox to latest version

### DISCUSSION THREAD

**Discussion by noor-tg** [feathersjs/feathers#3347] [General]
Hi,
It will be easier to use `Type.Composite` to make complicated schema with the latest version. this function does not exist on older versions of sinclair/typebox package.
so if you can update `@feathers/typbox` dependancy `sinclair/typbox` to latest version. or make it peer dependency so it can be updated freely later when need with new features.

thank you for the awesome framework.

**Comment by marshallswain** [comment]
This is on the radar.  There's a bigger issue for us to handle in order to make it possible.  Sinclair has been very helpful as much as possible, and even with the extra help, this update will create a breaking change.  But we're working on it.  :grin:
